Tests/Realm.Tests/Sync/MongoClientTests.cs

```diff
@@ -2173,6 +2261,47 @@ private async Task<MongoClient.Collection<Foo>> GetCollection()
             return collection;
         }
 
+        private async Task<MongoClient.Collection<T>> GetCollection<T>(string appConfigType = AppConfigType.Default)
+            where T : class, IRealmObject
+        {
+            var app = App.Create(SyncTestHelpers.GetAppConfig(appConfigType));
+            var user = await GetUserAsync(app);
+
+            // Use sync to create the schema/rules
+            SyncConfigurationBase config = appConfigType == AppConfigType.FlexibleSync ? GetFLXIntegrationConfig(user) : GetIntegrationConfig(user);
+
+            config.Schema = GetTransitiveSchema(typeof(T), new()).Values.ToArray();
```

Comment:
How come we're trying to limit the schema here? Is it to improve the speed of the test?